internal/compression/compression.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compression
+
+// The IANA names for supported compression algorithms
```

Comment:
I was thinking we could also have caches here for compressors/decompressors so that we aren't creating them over and over again for each test case. Not sure if that's overkill or not.

Reply:
I don't think these are expensive enough to create to worry about caching. A better place to cache, once the client is processing multiple RPC requests (instead of just one), is to probably cache entire RPC clients, keyed by target URL and options.